test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -125,12 +125,12 @@ TEST_F(YacGateTest, YacGateSubscriptionTest) {
 
   // verify that yac gate emit expected block
   auto gate_wrapper = make_test_subscriber<CallExact>(gate->on_commit(), 1);
-  gate_wrapper.subscribe([this](auto block) {
-    ASSERT_EQ(block, expected_block);
+  gate_wrapper.subscribe([this](auto commit) {
+    ASSERT_EQ(commit.block, expected_block);
```

Comment:
Also, I think, you can check the added `CommitType` here and in all changes places as well

Reply:
I think that will ruin the unit testing aspect since we only want to test one thing in each test.